spring-boot-autoconfigure-data-aerospike/src/main/java/org/springframework/boot/autoconfigure/aerospike/AerospikeProperties.java

```diff
@@ -94,15 +94,17 @@
      */
     private Boolean failIfNotConnected = true;
 
-    private ReadPolicyDefault read = new ReadPolicyDefault();
+    private PolicyDefault read = new PolicyDefault();
 
     private WritePolicyDefault write = new WritePolicyDefault();
 
+    private BatchPolicyDefault batch = new BatchPolicyDefault();
+
     /**
      * For more details on each option see corresponding field documentation in {@link com.aerospike.client.policy.Policy}.
      */
     @Data
-    public static class ReadPolicyDefault {
```

Comment:
Please leave ReadPolicyDefault class for configuring read, as using PolicyDefault for read-configuration is not readable and will not give an ability to add properties in the future without breaking someone who is using AerospikeProperties directly.

Reply:
Created separate ReadPolicyDefault for read config.